client/app/bundles/course/leaderboard/components/tables/LeaderboardTable.tsx

```diff
@@ -53,14 +72,27 @@ const styles = {
   avatar: {
     maxWidth: '250px',
     wordWrap: 'break-word',
-    display: 'inline-flex',
+    display: 'flex',
     alignItems: 'center',
     minWidth: '150px',
   },
 };
 
 const LeaderboardTable: FC<Props> = (props: Props) => {
   const { data, id: tableType } = props;
+  const tabletView = !useMediaQuery('(min-width:600px)');
```

Comment:
Great! Just wondering, why not use max-width instead?

Reply:
From what I saw, most usecases online use min-width, which was also what is used in [MUI's documentation](https://mui.com/material-ui/react-use-media-query/). But I think because their usecase is to check whether the width is greater than the width stated, but my usecase is for less than, so I think I will change to max-width so I don't have to negate. Will push new version